editor/gui/touch_actions_panel.cpp

```diff
@@ -181,4 +230,11 @@ TouchActionsPanel::TouchActionsPanel() {
 	delete_button = _add_new_action_button("", TTR("Delete"), Key::KEY_DELETE);
 	undo_button = _add_new_action_button("ui_undo", TTR("Undo"));
 	redo_button = _add_new_action_button("ui_redo", TTR("Redo"));
+	cut_button = _add_new_action_button("ui_cut", TTR("Cut"));
+	copy_button = _add_new_action_button("ui_copy", TTR("Copy"));
+	paste_button = _add_new_action_button("ui_paste", TTR("Paste"));
```

Comment:
```suggestion
	cut_button = _add_new_action_button("ui_cut", TTRC("Cut"));
	copy_button = _add_new_action_button("ui_copy", TTRC("Copy"));
	paste_button = _add_new_action_button("ui_paste", TTRC("Paste"));
```
Same should apply to existing buttons.